src/googleAnalytics.js

```diff
@@ -11,6 +11,12 @@ const trackGoogleAnalyticsEvent = (eventParams) => {
   }
 
   trackCustomEvent(eventParams);
+
+  if (!isUndefined(window.gtag)) {
+    window.gtag('event', 'pageview', {
+      'dimension1': `'${currentlyExecutingGitBranch()}'`,
```

Comment:
Worth naming this to 'git_branch' or similar?

Reply:
I suspect it can be deleted. I can't remember why that's there. I think I was debugging in prod. For the sake of this PR, I just want to cut across with feature parity. The deletion should be a separate step.